rc/filetype/zig.kak

```diff
@@ -57,13 +57,13 @@ add-highlighter shared/zig/code/ regex '\b(?:const|var|extern|packed|export|pub|
 add-highlighter shared/zig/code/ regex '\b(?:struct|enum|union|error|opaque)\b' 0:attribute
 
 # statement
-add-highlighter shared/zig/code/ regex '\b(?:break|return|continue|asm|defer|errdefer|unreachable|try|catch|async|noasync|await|suspend|nosuspend|resume)\b' 0:keyword
+add-highlighter shared/zig/code/ regex '\b(?:break|return|continue|asm|defer|errdefer|unreachable|try|catch||suspend|nosuspend|resume)\b' 0:keyword
```

Comment:
`catch||suspend`, is `""` a keyword now?

Reply:
Oopsie, fixed, sorry about that.